constellations/orion/src/repos/account_repo.go

```diff
@@ -177,20 +176,123 @@ func (acc *accountRepo) Update(id uint, account domains.Account) error {
 
 func (acc *accountRepo) Delete(id uint) error {
```

Comment:
Can we get some docs up here letting us know a summary of what's going on here?
This one delete operation is going to affect MANY tables in some way, it would be good to summarize them here